sdk/eventhub/event-hubs/README.md

```diff
@@ -79,7 +79,8 @@ The following sections provide code snippets that cover some of the common tasks
 
 - [Inspect an Event Hub](#inspect-an-event-hub)
 - [Publish events to an Event Hub](#publish-events-to-an-event-hub)
-- [Consume events from an Event Hub](#consume-events-from-an-event-hub)
+- [Consume events from an Event Hub partition](#consume-events-from-an-event-hub-partition)
+- [Consume events from all partitions of an Event Hub](#consume-events-from-all-partitions-of-an-event-hub)
```

Comment:
How about `Consume events using an Event Processor` ?

That will give an idea that this sample is very much different from the previous one